production/helm/loki/templates/gateway/deployment-gateway-enterprise.yaml

```diff
@@ -101,7 +101,7 @@ spec:
             {{ toYaml .Values.enterpriseGateway.extraVolumeMounts | nindent 12 }}
             {{- end }}
           ports:
-            - name: http-metrics
+            - name: http
```

Comment:
when the loki/gel image is run in mode `gateway`(`-target=gateway`), it exposes the metrics under `/metrics` url ....